### DISCUSSION TITLE
Wayback module option to visit archived sites

### DISCUSSION THREAD

**Discussion by liquidsec** [blacklanternsecurity/bbot#724] [Ideas]
Occasionally we may want to visit the actual archived pages as opposed to the live pages. This would be an option that would be disabled by default.

**Comment by liquidsec** [comment]
probably best to wait until after presets are enabled

**Comment by Sh4d0wHunt3rX** [comment]
That would be awesome if possible to add this. Right now I'm using waymore:
https://github.com/xnl-h4ck3r/waymore

It supports:
Wayback Machine (web.archive.org)
Common Crawl (index.commoncrawl.org)
Alien Vault OTX (otx.alienvault.com)
URLScan (urlscan.io)
Virus Total (virustotal.com)

**Comment by Sh4d0wHunt3rX** [comment]
Also, that would be great if possible to add ability to fetch old robots.txt contents, I'm using robofinder for this currently:
https://github.com/0xSpidey/robofinder

**Comment by Sh4d0wHunt3rX** [comment]
That would be interesting not only extract URL from wayback, but also the parameters and send them to other modules for checking reflecting. Similar to:
https://github.com/devanshbatham/ParamSpider

**Reply by TheTechromancer** [reply]
I think @liquidsec is already working on this

**Comment by Sh4d0wHunt3rX** [comment]
https://twitter.com/palewire/status/1770093088771285377
![image](https://github.com/blacklanternsecurity/bbot/assets/15929497/9cd20e3a-465e-484d-9a27-2ab8631e6791)

**Comment by liquidsec** [comment]
I want to revisit this, this is a great idea

**Comment by liquidsec** [comment]
this is implemented ! :)
